payment/template.yaml

```
@@ -258,4 +258,391 @@ Resources:
+                "region": "${AWS::Region}",
+                "title": "OnCreated Latency"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 10,
+              "y": 4,
+              "width": 10,
+              "height": 3,
+              "properties": {
+                "metrics": [
+                  ["AWS/Lambda", "Duration", "FunctionName", "${OnCompletedFunction}", {"color": "#9ccc65", "label": "p50"}],
+                  ["...", {"stat": "p90", "color": "#ffee58", "label": "p90"}],
+                  ["...", {"stat": "p99", "color": "#ef5350", "label": "p99"}]
+                ],
+                "view": "singleValue",
+                "period": 86400,
+                "stacked": false,
+                "stat": "p50",
+                "region": "${AWS::Region}",
+                "title": "OnCompleted Latency"
+              }
+            },
+
+            {
+              "type": "text",
+              "x": 0,
+              "y": 7,
+              "width": 24,
+              "height": 1,
+              "properties": {
+                "markdown": "\n# Traffic\n"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 0,
+              "y": 8,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "metrics": [
+                  ["AWS/Lambda", "Invocations", "FunctionName", "${OnCreatedFunction}", {"color": "#29b6f6", "label": "CreatePayment"}],
+                  ["AWS/Lambda", "Invocations", "FunctionName", "${OnCompletedFunction}", {"color": "#66bb6a", "label": "CompletePayment"}],
+                  ["AWS/Lambda", "Invocations", "FunctionName", "${OnFailedFunction}", {"color": "#ec407a", "label": "CancelPayment"}],
+                  ["AWS/Lambda", "Invocations", "FunctionName", "${OnModifiedFunction}", {"color": "#b300b3", "label": "ModifyPayment"}],
+                  ["AWS/Lambda", "Invocations", "FunctionName", "${ValidateFunction}", {"color": "#ffa726", "label": "ValidatePayment"}]
+                ],
+                "view": "timeSeries",
+                "period": 60,
+                "stacked": false,
+                "stat": "Sum",
+                "region": "${AWS::Region}",
+                "title": "Requests"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 12,
+              "y": 8,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "metrics": [
+                  ["ecommerce.payment", "paymentCreated", "environment", "${Environment}", {"color": "#29b6f6", "label": "Payment Created"}],
+                  ["ecommerce.payment", "paymentCompleted", "environment", "${Environment}", {"color": "#66bb6a", "label": "Payment Completed"}],
+                  ["ecommerce.payment", "paymentCanceled", "environment", "${Environment}", {"color": "#ef5350", "label": "Payment Canceled"}]
```

To stay consistent with the previous comments:

```suggestion
                  ["ecommerce.payment", "paymentProcessed", "environment", "${Environment}", {"color": "#66bb6a", "label": "Payment Processed"}],
                  ["ecommerce.payment", "paymentCancelled", "environment", "${Environment}", {"color": "#ef5350", "label": "Payment Cancelled"}]
```